example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -494,17 +494,18 @@ describe('When there are granule differences and granule reconciliation is run',
       expect(extraDbUris).toContain(extraFileUri);
     });
 
-    it('generates a report showing number of collections that are in both Cumulus and CMR', () => {
+    it('generates a filtered report showing requested collections that are in both Cumulus and CMR', () => {
       if (beforeAllFailed) fail(beforeAllFailed);
       // MYD13Q1___006 is in both Cumulus and CMR
-      expect(report.collectionsInCumulusCmr.okCount).toBeGreaterThanOrEqual(1);
+      expect(report.collectionsInCumulusCmr.okCount).toBe(1);
```

Comment:
Good point, I don't know why that wasn't tested before. Ah, that's because collections that match are only described as "ok"  So the output from the reconciliation report looks like this:

```
  "collectionsInCumulusCmr": {
    "okCount": 1,
    "onlyInCumulus": [
      "collection-name-fc11ec949c___collection-version-de1bbbb83e"
    ],
    "onlyInCmr": [
      "L2_HR_PIXC___1"
    ]
  },
```
And the okCount used to be > 1 but now it's exactly one.  There's no information about that collections name available.